.github/ISSUE_TEMPLATE/broken-website.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Broken website
+about: Creates a broken website report about Darkreader
+title: "[Broken Website] "
+labels: Broken Website
+assignees: ''
+
+---
+
+<!--
+Thank you for taking the time to report a broken website.
+Please make sure their aren't existing issue's about this broken website.
```

Comment:
I'm not an English speaker, but I think it should be `Please make sure there is no existing issue...`.

Reply:
Updated the grammarrr